.github/kubeval.sh

```diff
@@ -1,11 +1,11 @@
 #!/bin/bash
-set -euxo pipefail
+set -uxo pipefail
```

Comment:
Removed `set -e` because the git commands might fail making the whole script to fail.

The `helm template` and kubeval checks at the end only execute when any of the `Chart.yaml` files was modified, which is not the case of this patch.

I also made the grep and sed into easier to read alternatives.

Reply:
I see. Yeah, seems great, thanks 🙏